src/components/results/shortcircuit/shortcircuit-analysis-result-tab.tsx

```diff
@@ -145,6 +166,13 @@ export const ShortCircuitAnalysisResultTab: FunctionComponent<ShortCircuitAnalys
                     <Tab label={<FormattedMessage id={'Results'} />} />
                     <Tab label={<FormattedMessage id={'ComputationResultsLogs'} />} />
                 </Tabs>
+                {resultOrLogIndex === RESULTS_TAB_INDEX && tabIndex === ShortCircuitAnalysisResultTabs.ALL_BUSES && (
+                    <GlobalFilterSelector
+                        onChange={handleGlobalFilterChange}
+                        filters={globalFilterOptions}
+                        filterableEquipmentTypes={filterableEquipmentTypes}
```

Comment:
You should disable the selection of invalid filters, like in the other results pages : 
```suggestion
                        filterableEquipmentTypes={filterableEquipmentTypes}
                        genericFiltersStrictMode={true}
```
(will hide filters types that are not part of filterableEquipmentTypes)

Reply:
If the global filter is not applied, you may have made the mistake to use an old study-server?

Reply:
<img width="1384" height="322" alt="image" src="https://github.com/user-attachments/assets/dfd3ad22-b5ed-4ad0-8bd6-aef6aaba41e1" />
The global filter is not always near the tabs

Reply:
Yes but this is a bad thing. It should be harmonized according to Stephane decision.

Reply:
<img width="1600" height="604" alt="image" src="https://github.com/user-attachments/assets/f90204f0-2e7d-40f9-9806-211af19d5b82" />

Reply:
It's okay now :p